wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,723 @@
+- External links
+- Tags
+
+## Front matter
+
+Front matter must be placed at the very top of the file. It is written in [YAML](https://en.wikipedia.org/wiki/YAML#Example "YAML Wikipedia article") and describes additional information about the article. This must be surrounded by three hyphens (`---`) on the lines above and below it, and an empty line must follow it before the title heading.
+
+### Articles that need help
+
+The `needs_cleanup` tag may be added to articles that need rewriting or formatting help. It is also acceptable to open an issue on GitHub for this purpose.
+
+*Notice to translators: English articles with this tag should be avoided. This tag applies when the translation needs its own clean up and must not be blindly copied over from the English article.*
+
+For example:
+
+```yaml
+needs_cleanup: true
+```
+
+When marking articles with this, [comments](#comments) should accompany this to explain what needs to be cleaned.
+
+### Outdated articles
+
+*Notice to translators: Avoid translating English articles with this tag.*
+
+Translated articles that are outdated must use the `outdated` mark when the English variant is updated. English articles may also become outdated when the content they contain is misleading or no longer relevant. For example:
+
+```yaml
+outdated: true
+```
+
+*Note: If the English article has this tag, all translations must have this tag as well.*
+
+When marking articles with this, [comments](#comments) should accompany this to explain what content is outdated.
+
+### Tagging articles
+
+Tags help the website's search engine query articles better. Tags should be written in the same language as the article and include the English's list of tags, and prefer lowercase letters where applicable.
+
+For example, an article called "Skinning tutorial" might include these tags:
+
+```yaml
+tags:
+  - guide
+  - how to
+  - skins
+```
+
+### Translations without reviews
+
+Sometimes, translations are added to the wiki without review from other native speakers of the language. In this case, the `no_native_review` mark is added to let future translators know that it may need to be checked again. For example:
+
+```yaml
+no_native_review: true
+```
+
+*Note to translators: Wiki maintainers will determine and apply this mark prior to merging.*
+
+## Article naming
+
+*See also: [Folder names](#folder-names) and [Titles](#titles)*
+
+Article titles should be singular. See [Wikipedia's naming conventions article](https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(plurals) "Wikipedia") for more details.
+
+Article titles should match the folder name it is in (spaces may replace underscores (`_`) where appropriate). If the folder name changes, the article title should be changed to match it and vise versa. Some exceptions include:
+
+- Contest articles - must use abbreviations, acronyms, or initialisms
+- Tournament articles - must use abbreviations, acronyms, or initialisms
```

Comment:
consider changing to parantheses and not spaced hyphen (which should be en dash anyway `—`)